### ISSUE TITLE
Handle core vocabulary verb: Put

### ISSUE THREAD

**Issue by denizbeser** [isi-vista/adam#199]
Put (~3000): Almost always take an NP argument (~2800), almost all with PP/ADVP

Put: V location

**Comment by rgabbard** [comment]
Based on what @charles-yang-upenn says, I would do the following three templates:

```
X puts Y in Z
X puts Y on Z
```

Make a new issue for _X puts Y down_; put it in the backlog

**Comment by mitchmarcus** [comment]
I agree that put (as opposed to "put down", which is a different verb) takes three arguments - one of the few that do.  But any loc will do as the 3rd arg "_put x <under,in,on,near,above,below, ..>Y)_, so I would say they're all one template.

**Comment by rgabbard** [comment]
@mitchmarcus : You are correct they should be one template; I just have't implemented the iterating over a set of `GOAL`s yet (that's coming this evening), so for now Deniz is doing those two as a placeholder.
